recipe/build.sh

```diff
@@ -15,7 +15,7 @@ if [[ "${target_platform}" == "linux-aarch64" ]] || [[ "${target_platform}" == "
 fi
 
 pushd build-target
-cmake ${CMAKE_ARGS} \
+cmake ${CMAKE_ARGS} --trace-expand \
```

Comment:
Added to see if we can get some more details about how the CMake configure step is going as well as see if `libcudart` is being picked up here somewhere